src/cases/components/role-exclusions/role-exclusions.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, OnInit} from '@angular/core';
+import { RoleExclusion } from '../../models/role-exclusions/role-exclusion.model';
+
+@Component({
+    selector: 'exui-role-exclusions',
+    templateUrl: './role-exclusions.component.html'
+  })
+
+export class RoleExclusionsComponent implements OnInit {
+  @Input()
+  public exclusions: RoleExclusion [];
+
+  @Input()
+  public allowDelete: boolean = false;
+
+  ngOnInit(): void {
+  }
+  public deleteExclusion(exclusion: RoleExclusion) {
+    console.log(exclusion);
```

Comment:
Do we need this print for now?

Reply:
Yes, Thats the way @sreekanthpuligadda  can confirm the click event is selected until we have Delete functionality

Reply:
ok thanks